modules/gce_project/main.tf

```diff
@@ -236,7 +236,7 @@ resource "google_compute_firewall" "deny_target_ip" {
   project = "${var.project}"
 
   # highest priority
-  priority = "0"
+  priority = "1000"
```

Comment:
The [docs](https://www.terraform.io/docs/providers/google/r/compute_firewall.html#priority) mention that 0 is the highest priority and 1000 is the default. What are we achieving with this change?

Reply:
This is another one that I addressed via commit comment but forgot to update the PR description 😅.  The documented highest priority of `0` appears to be either untrue or perhaps terraform is doing something special with the zero value such that repeat terraform plans wanted to change the value to `1000`.